transform/mattermost-analytics/models/intermediate/web_app/platform/int_mm_telemetry_prod_performance_events.sql

```diff
@@ -0,0 +1,14 @@
+{{
+    config({
+        "materialized": "incremental",
+        "cluster_by": ['received_at_date'],
+        "snowflake_warehouse": "transform_l"
```

Comment:
@sri-byte , do we have to set this here, specifically for this model? Or should we set it at a higher level for the whole set of performance models?

Reply:
I don't think we need this for RC model based on the volume of data, and another point is we don't need this long term, since the incremental strategy for this is `append`. I plan to remove this once the tables are created.

Reply:
I was just checking this and it's taking a long time. The default is `transform_xs`, so it makes sense to override it for this slow model. The query is rather simple, ~80% of the time is sorting.